content/docs/command-reference/params/index.md

```diff
@@ -143,6 +143,98 @@ $ dvc run -n train -d logs/ -o users.csv \
           python train.py
 ```
 
+## Examples: Python parameters file
+
+Consider this parameters file in Python format, named `params.py`:
+
+```python
+IS_BOOL: bool = True
+CONST = 5
+
+# All standard variable types are supported
+FLOAT = 0.001
+STR = 'abc'
+DICT = {
+    "a": 1,
+    "b": 2
+}
+LIST = [1, 2, 3]
+SET = {4, 5, 6}
+TUPLE = (10, 100)
+NONE = None
+
+
+# It is possible to retrieve either class constants
+# or own variables defined in __init__
+class TrainConfig:
+    EPOCHS = 70
+
+    def __init__(self):
+        # TrainConfig.layers param will be 9
+        self.layers = 5
+        self.layers = 9
+        # TrainConfig.foo will NOT be found because the complex expression
+        self.foo = 1 + 2
+        # TrainConfig.bar will NOT be found
+        bar = 1
```

Comment:
```suggestion
        self.layers = 5
        self.layers = 9     # TrainConfig.layers param will be 9
        self.sum = 1 + 2    # Will NOT be found due to the expression
```
(The last one is simply not defined in the class so kind of a Python thing.)

Reply:
Why can't DVC find vars that need evaluation though? Isn't the whole params file interpreted by Python first @aandrusenko @efiop? (Just curious) Thanks

Reply:
Nope, parameters are parsed using the ast module, it parses complex expressions into a complex structure that I did not support

Reply:
I see, thanks.